internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -45,14 +45,20 @@ func (d *DockerfileHybrid) GetInput() (input.Input, error) {
 	return d.Input, nil
 }
 
-const dockerFileHybridAnsibleTmpl = `FROM ansible/ansible-runner
+const dockerFileHybridAnsibleTmpl = `FROM ansible/ansible-runner:1.2
 
 RUN yum remove -y ansible python-idna
 RUN yum install -y inotify-tools && yum clean all
 RUN pip uninstall ansible-runner -y
 
-RUN pip install --upgrade setuptools
-RUN pip install ansible ansible-runner openshift kubernetes ansible-runner-http idna==2.7
+RUN pip install --upgrade setuptools==41.0.1
+RUN pip install "urllib3<1.25,>=1.23"
+RUN pip install ansible==2.7.10 \
+	ansible-runner==1.2 \
+	ansible-runner-http==1.0.0 \
+	idna==2.7 \
+	kubernetes==9.0.0 \
```

Comment:
openshift `0.8.x` is only testing against kubernetes `8.x.x`, so we should probably do `'kubernetes >= 8.0.0, < 9.0.0'`